cli_tools/daisy/main.go

```diff
@@ -38,6 +39,7 @@ var (
 	zone               = flag.String("zone", "", "zone to run in, overrides what is set in workflow")
 	variables          = flag.String("variables", "", "comma separated list of variables, in the form 'key=value'")
 	print              = flag.Bool("print", false, "print out the parsed workflow for debugging")
+	printPerf          = flag.Bool("print_perf", false, "print out the perf profile for debugging")
```

Comment:
Use "performance" in descriptions/help texts

Also, is "for debugging" actually correct? We can use this for workflow optimization etc. not only debugging. I would remove that part.

Reply:
Sounds good